ansible/roles/integration_tests/tasks/check_bundle_existence_in_index_image.yml

```diff
@@ -0,0 +1,68 @@
+---
+- name: Verify that bundle was released to index image
+  vars:
+    bundle_name: "{{ operator_package_name }}.v{{ operator_bundle_version }}"
+    tmp_folder: /tmp/check_bundle/
+  block:
+
+    - name: Verify that bundle was released to index image
+
+      block:
+        - name: Create directory for image data
+          ansible.builtin.file:
+            path: "{{ tmp_folder }}"
+            state: directory
+            mode: 0777
+
+        - name: Create directory for registry credentials
+          ansible.builtin.file:
+            path: "{{ tmp_folder }}registry_certs/"
```

Comment:
It is kind of weird to see no slash between `{{ tmp_folder }}` and rest of the path. Wouldn't it be better to remove the slash from `tmp_folder` and explicitly add it to the places where you concatenate paths?